scripts/run-on-error.js

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
+const execa = require('execa')
+const { argv } = require('process')
+
+const [, , npmScript, npmScriptOnError] = argv
+
+// Run a npm script. If that script fails, another npm script is run.
+// We use this for example with ESLint and Prettier to be able to fail if
+// anything should be autofixed, while still autofixing it. Those tools do not
+// provide good CLI flags for this.
+const runOnError = async function() {
+  const { failed, exitCode = DEFAULT_ERROR_EXIT_CODE } = await runNpmScript(npmScript)
```

Comment:
`exitCode` can be `undefined` when the script has failed due to a standard stream error or a signal termination.